.github/workflows/code-checks.yml

```diff
@@ -34,7 +34,8 @@ jobs:
       run: |
         pip install torch==1.11 --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
         python ./requirements/pytorch/adjust-versions.py requirements/pytorch/extra.txt
-        pip install -r requirements/pytorch/devel.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+        pip install -e . --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+        pip install "mypy==0.920"
```

Comment:
This shouldn't be pinned.

Reply:
Why not pin? Some versions may have a different interpretation, and we experienced it past...

Reply:
I think it should be pinned within one of the requirements files. Otherwise, it will lead to different versions between local dev envs and our CI.